include/Makefile.am

```diff
@@ -68,6 +69,11 @@ odpdrvspecinclude_HEADERS = \
 		  odp/drv/spec/std_types.h \
 		  odp/drv/spec/sync.h
 
+frameworksmodularspecincludedir= $(includedir)/frameworks/modular/spec
+frameworksmodularspecinclude_HEADERS = \
+		  frameworks/modular/spec/module.h \
+		  frameworks/modular/spec/list.h
```

Comment:
list.h should not be part of the 'spec' directory. Should be part of framework source directory.

Ideally, we should split module.h to contain only the macros that need to be exposed to subsystems and move the rest to framework source directory.

Reply:
true, but this cannot be done in this patch.